Dockerfile.gcc

```diff
@@ -104,9 +104,10 @@ RUN --mount=type=cache,target=/root/.cache/ccache,id=kernel-ccache-${TARGETARCH}
         INSTALL_MOD_PATH=/tmp/kernel-modules \
     && ccache -s
 
-ADD https://github.com/mikem-zed/zfs.git#eve-zfs-2.1.12 /tmp/zfs
 WORKDIR /tmp/zfs
-
+ADD https://github.com/openzfs/zfs/tarball/zfs-2.2.2/ zfs.tgz
```

Comment:
@andrewd-zededa , you can pass the git path (not the tarball) to the ADD command so you don't need this additional step to extract the tarball... changing the ADD to `ADD https://github.com/openzfs/zfs.git#zfs-2.2.2 /tmp/zfs` is enough...

Reply:
Ok, i'll make the change

Reply:
Hi @rene this PR has been updated to your suggestion above.